### PR TITLE
2993 cy data fed prod

### LINKED ISSUE #2993: Updated CY Federal Production
- [x] Check file format and known data issues

- [x]  Update visualizations

### PR DESCRIPTION
**Pull request by master12**
Fixes #2993 

[:sunglasses: PREVIEW](https://federalist-proxy.app.cloud.gov/preview/onrr/doi-extractives-data/2993-cy-data-fed-prod/explore/AK/#federal-production)

Changes proposed in this pull request:

- Updated years.yml
- Updated site data
-

### CODE REVIEW

**Comment by master12** [comment]
@mentastc @mcharg Can you 2 review the updated data please

File: _data/federal_county_production/AL.yml [hunk 1]
```diff
@@ -5,46 +5,37 @@
...
         '2016': 22
-'01007':
-  name: Bibb County
-  products:
-    Clay (tons):
-      name: Clay
-      units: tons
-      volume:
-        '2007': null
+        '2017': 17
```
Comment (mcharg) [inline _data/federal_county_production/AL.yml]: This is the clay in Bibb County that I don't see on the site or in the spreadsheet.

Comment (master12) [reply _data/federal_county_production/AL.yml]: This says it was removed

**Comment by mentastc** [comment]
Looks good I completed my review.

**Pull request merged**